rules/establish-the-world/rule.md

```diff
@@ -0,0 +1,44 @@
+---
+type: rule
+title: Do you establish the world for your audience?
+uri: establish-the-world
+authors:
+  - title: Jonty Gardner
+  - url: https://ssw.com.au/people/jonty-gardner
+created: 2023-10-04T06:22:53.243Z
+guid: f1dac5ed-a422-4e19-bf72-f6c637ba0a23
+---
+## 🎥 Do you establish the world for your audience?
```

Comment:
Hey @jontygardner 

No reason to repeat the title here... 

1. Please delete this heading

Reply:
Done